tests/SpecTests/ReadWriteConcernSpecTest.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace MongoDB\Tests\SpecTests;
+
+use stdClass;
+use function basename;
+use function dirname;
+use function file_get_contents;
+use function glob;
+
+/**
+ * @see https://github.com/mongodb/specifications/tree/master/source/read-write-concern
+ */
+class ReadWriteConcernSpecTest extends FunctionalTestCase
```

Comment:
This entire class/file appears to be identical to CrudSpecTest. The only difference I could see was the argument to glob() call, giving a different path. Would it not be better to reuse this code somehow and only specify the different path to where the spec test files are stored?

Reply:
Yes, there is a lot of duplication in these tests. I plan to refactor them to extract common functionality and build extension points, since some of the spec tests use a slightly different format.

Reply:
The unified test runner project might also allow us to consolidate everything into a common `UnifiedSpecTest.php` file, and then have extending classes only implement a data provider to pull in relevant JSON files.

Reply:
:+1: